app/src/main/java/com/bashkevich/androidfundamentals/model/EntityMapper.kt

```diff
@@ -0,0 +1,32 @@
+package com.bashkevich.androidfundamentals.model
+
+import com.bashkevich.androidfundamentals.model.dto.ActorDto
+import com.bashkevich.androidfundamentals.model.dto.GenreDto
+import com.bashkevich.androidfundamentals.model.dto.MovieDto
+import com.bashkevich.androidfundamentals.model.entity.Actor
+import com.bashkevich.androidfundamentals.model.entity.Genre
+import com.bashkevich.androidfundamentals.model.entity.Movie
+
+object EntityMapper {
+    fun convertToMovieFromDto(movieDto: MovieDto): Movie {
+
+        return Movie(
+            movieDto.id,
+            movieDto.title,
+            movieDto.overview,
+            movieDto.posterPicture?.let { it },
+            movieDto.backdropPicture?.let { it },
```

Comment:
Same

Reply:
sometimes backdrop was null, and that is why I added null check.

Reply:
You don't really checking anything here - this statement will still return null if `backdropPicture` is null